test-engineering/contract-tests/client/tests/kinto.py

```diff
@@ -0,0 +1,98 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from typing import Set
+
+import requests
+from pydantic import BaseModel
+from requests import Response as RequestsResponse
+
+
+class KintoEnvironment(BaseModel):
+    """Class that holds information about Kinto environment variables."""
+
+    server: str
+    bucket: str
+    collection: str
+
+
+class KintoAttachment(BaseModel):
+    """Class that holds information about an attachment in Kinto."""
+
+    filename: str
+    filecontent: bytes
+    mimetype: str
+    icon_ids: Set[str]
+
+
+class KintoRecord(BaseModel):
+    """Class that holds information about a record in Kinto."""
+
+    record_id: str
+    attachment: KintoAttachment
+
+
+def get_record(environment: KintoEnvironment, record_id: str) -> RequestsResponse:
+    """Get attachment information from Kinto for the given record ID."""
+
+    url: str = (
+        f"{environment.server}/v1/"
+        f"buckets/{environment.bucket}/"
+        f"collections/{environment.collection}/"
+        f"records/{record_id}"
+    )
+    response: RequestsResponse = requests.get(url)
+    response.raise_for_status()
+    return response
```

Comment:
Hmm 🤔 

I'd expect a method named `kinto.get_record()` to return a Kinto record object, but we're returning the `requests.Response` object directly here. Should we move the `response.json()` call in `fetch_icon_url` here and return a `Dict` with the response body or even a model instance?

From what I can tell that's the only reason why consumers (`conftest.py`) of our kinto client (`get_record()`, `upload_attachment()`, `upload_icons()`) import from `requests`.

Reply:
change applied: https://github.com/mozilla-services/merino/pull/409/commits/84119c1833d8ac1e09a2362ad84070d72307af42

I think that's a fair assumption, I've made an adjustment.